src/lumigo_tracer/parsers/event_parser.py

```diff
@@ -82,10 +94,58 @@ def parse(event) -> Dict:
         return new_event
 
 
+class SNSHandler(EventParseHandler):
+    @staticmethod
+    def is_supported(event) -> bool:
+        if (
+            isinstance(event, dict)
+            and event.get("Records")  # noqa
+            and event.get("Records", [{}])[0].get("EventSource") == "aws:sns"  # noqa
+        ):
+            return True
+        return False
+
+    @staticmethod
+    def parse(event) -> Dict:
+        new_sns_event: OrderedDict = OrderedDict()
+        new_sns_event["Records"] = []
+        # Add order keys
+        for rec in event.get("Records"):
+            new_sns_record_event: OrderedDict = OrderedDict()
+            for key in SNS_KEYS_ORDER:
+                new_sns_record_event[key] = rec["Sns"].get(key)
```

Comment:
don't you prefer to check first if `key in res["Sns"]`?
Otherwise we will show the customer a field that wasn't there beforehand....